scripts/run_mypy_checks_test.py

```diff
@@ -68,12 +71,39 @@ def mock_popen_failure(
             run_mypy_checks, 'EXCLUDED_DIRECTORIES',
             ['dir1/', 'dir2/'])
 
-        def mock_install_mypy_prerequisites_success():
+        def mock_install_mypy_prerequisites_success(ci): # pylint: disable=unused-argument
```

Comment:
It's best to avoid disabling pylint if at all possible. In this case, I think you can change the argument name to be `unused_ci` instead

Reply:
Done. Thanks!